easy-ui-react/src/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { ReactNode } from "react";
+import { Key } from "react-aria";
+import { AriaLabelingProps } from "@react-types/shared";
+import { ToggleButtonGroup, ToggleButton } from "react-aria-components";
+import { classNames, getComponentThemeToken } from "../utilities/css";
+import { ThemeColorAliases } from "../types";
+import { Text } from "../Text";
+
+import styles from "./RadioButtonGroup.module.scss";
+
+export type RadioButtonGroupButtonProps = {
+  /**
+   * The label for the radio button.
+   */
+  children: ReactNode;
+  /**
+   * Whether the radio button is disabled or not.
+   */
+  isDisabled?: boolean;
+  /**
+   * An identifier for the item in selectedKeys.
+   */
+  id: string;
+};
+
+function RadioButtonGroupButton(props: RadioButtonGroupButtonProps) {
+  const { children, isDisabled, id } = props;
+
+  return (
+    <ToggleButton
+      className={classNames(styles.RadioButtonGroupButton)}
+      isDisabled={isDisabled}
+      id={id}
+    >
+      <Text variant="small_button">{children}</Text>
+    </ToggleButton>
+  );
+}
+
+export type RadioButtonGroupProps = AriaLabelingProps & {
+  /**
+   * Color for the selected button in the group.
+   * @default "primary.500"
+   */
+  color?: ThemeColorAliases;
+  /**
+   * Whether single or multiple selection is enabled.
+   * @default "single"
+   */
+  selectionMode?: "single" | "multiple";
+  /**
+   * Whether the collection allows empty selection.
+   */
+  disallowEmptySelection?: boolean;
+  /**
+   * 	The currently selected keys in the collection (controlled).
+   */
+  selectedKeys?: Set<Key>;
+  /**
+   * The initial selected keys in the collection (uncontrolled).
+   */
+  defaultSelectedKeys?: string[];
```

Comment:
these should be `Iterable<Key>`:

```suggestion
  selectedKeys?: Iterable<Key>;
  /**
   * The initial selected keys in the collection (uncontrolled).
   */
  defaultSelectedKeys?: Iterable<Key>;
```